### PR TITLE
Moving Threads to a reusable modal

### PR DESCRIPTION
**Pull request by andrewxhill**
Primary goal was to help with the Nav mess of having it in the view stack

Ended up allowing me to clean up a few other parts of the UX

Don't love the bottom slide-up look yet, but ideally will replace the modal with a full screen swiss army knife that allows you to select contacts to add at the same time you create the thread

already lots of changes though, so breaking it off here and will continue as smaller parts

### CODE REVIEW

File: App/Navigation/AppNavigation/Primary/TabNavigator/index.tsx [hunk 1]
```diff
@@ -37,7 +37,7 @@ const nav = createBottomTabNavigator(
     },
     animationEnabled: false,
     swipeEnabled: false,
-    initialRouteName: 'Wallet'
+    initialRouteName: 'Threads'
```
Comment (asutula) [inline App/Navigation/AppNavigation/Primary/TabNavigator/index.tsx]: Just making sure you want this in there? Thought you wanted to highlight Wallet.

**Comment by andrewxhill** [comment]
okay, gtg

File: App/Containers/Wallet.js [hunk 2]
```diff
@@ -29,21 +26,18 @@ class Wallet extends React.PureComponent {
...
-    const accountBackupEnabled = params.storage && params.storage.enableWalletBackup.status
-    const pinEnabled = params.storage && params.storage.autoPinPhotos.status
     const photoBackupEnabled = params.storage && params.storage.enablePhotoBackup.status
 
     const headerLeft = (
       <TextileHeaderButtons>
         <Item
           title='backup'
-          iconName='cloud'
+          iconName={photoBackupEnabled ? 'cloud-checked' : 'cloud'}
```
Comment (asutula) [inline App/Containers/Wallet.js]: Nice. You can also set the color, just pass a `color` prop set to brandBlue or whatever. If you want.

Comment (andrewxhill) [reply App/Containers/Wallet.js]: actually didn't work... didn't dig deep into it but yeah, color param was there before and not working

**Comment by asutula** [comment]
Ok. Good enough for now

**Pull request merged**